programs/eosioc/main.cpp

```diff
@@ -1112,10 +1112,12 @@ int main( int argc, char** argv ) {
       auto accountPermissions = get_account_permissions(permissions);
 
       signed_transaction trx;
-      /* TODO: restore this function
-      transaction_emplace_serialized_actions(trx, contract, action, accountPermissions,
-                                                      result.get_object()["binargs"].as<bytes>());
-      */
+      eosio::chain::action act;
+      act.account = contract;
+      act.name = action;
+      act.authorization = accountPermissions;
+      act.data = result.get_object()["binargs"].as<bytes>();
+      trx.actions.push_back(act);
 
```

Comment:
Had same fix on my branch, but I added an `action` constructor:

```
      action( vector<permission_level> auth, account_name account, action_name name, const bytes& data )
            : account(account), name(name), authorization(move(auth)), data(data) {
      }
```

Also renamed command to `action` from `actions`

Reply:
I did the actions->action change, but I'll leave the lack of new ctor for now